packages/hdwallet-tally/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@shapeshiftoss/hdwallet-tally",
+  "version": "1.19.0",
+  "license": "MIT",
+  "publishConfig": {
+    "access": "public"
+  },
+  "main": "dist/index.js",
+  "source": "src/index.ts",
+  "types": "dist/index.d.ts",
+  "scripts": {
+    "build": "tsc --build",
+    "clean": "rm -rf dist tsconfig.tsbuildinfo",
+    "prepublishOnly": "yarn clean && yarn build"
+  },
+  "dependencies": {
+    "@metamask/detect-provider": "^1.2.0",
```

Comment:
is there a more vanilla way of doing this, rather than coupling tally to a metamask dependency?

Reply:
Yes and I have added the appropriate functions to remove the MM dependancy